metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala

```diff
@@ -109,6 +116,44 @@ case class ScalafixProvider(
     }
   }
 
+  private def scalafixConf: Option[Path] = {
+    val scalafixConfPath = userConfig().scalafixConfigPath
+      .getOrElse(workspace.resolve(".scalafix.conf"))
+    if (scalafixConfPath.isFile) Some(scalafixConfPath.toNIO)
+    else None
+  }
+
+  private def evaluateScalafix(
```

Comment:
Super nitpicky, but to me when I read this I think we are evaluating Scalafix. Can we instead do:
```suggestion
  private def scalafixEvaluate(
```

Reply:
Fixed!